string  readable_status
int32   status_code